test/distributed_exec.jl

```diff
@@ -661,7 +661,20 @@ catch ex
     @test typeof(ex.exceptions[1].ex) == ErrorException
     errors = map(x->x.ex.msg, ex.exceptions)
     @test collect(1:5) == sort(map(x->parse(Int, x), errors))
-end
+    # test start, next, and done
+    for (i, i_ex) in enumerate(ex)
+        @test i == parse(Int, i_ex.ex.msg)
+    end
+    # test showerror
+    err_strs = split(sprint(showerror, ex), "\n")
```

Comment:
Wouldn't it be simpler to replace this block with  
```
@test sprint(showerror, ex) == sprint(showerror, ex.exceptions[1]) * "\n\n...and 4 more exception(s).\n"
```
This way if the call stack changes, we do not have to come back and change the test here.